engine/menus/intro_menu.asm

```diff
@@ -999,8 +999,8 @@ StartTitleScreen:
 	ld b, SCGB_DIPLOMA
 	call GetSGBLayout
 	call UpdateTimePals
-	ld a, [wIntroSceneFrameCounter]
-	cp $5
+	ld a, [wTitleScreenSelectedOption]
+	cp 5
```

Comment:
These 5 values 0-4 should have constants corresponding to the `StartTitleScreen.dw` entries.

Reply:
Will do in either a different PR or after https://github.com/pret/pokecrystal/issues/748

Reply:
Don't wait for #748, we haven't settled on a macro, and once we do it'll be easier to apply when files already have constants defined at the tops.